Frameworks/AddressBook/ABContactInternal.mm

```diff
@@ -28,4 +28,18 @@ - (id)initWithContact:(WACContact*)contact {
     return self;
 }
 
++ (NSDate*)convertDate:(WACContactDate*)date {
+    NSCalendar* calendar = [NSCalendar currentCalendar];
+    NSDateComponents* dateComponents = [[NSDateComponents alloc] init];
+
+    // Grab the numerical values of the day/month/year,
+    // and create an NSDate to match the same day as the
+    // given NSDate.
+    dateComponents.day = [date.day integerValue];
+    dateComponents.month = [date.month integerValue];
+    dateComponents.year = [date.year integerValue];
```

Comment:
Windows.ApplicationModel.Contacts.ContactDate has some tips on integrating with actual dates:

1. If year has a value, set year to that value, else set year to 2004 (a leap year so every day is valid).
2. If month has a value, set month to that value, else set month to 1 (a month with every day valid).
3. if day has a value, set day to that value (day will be set to some value automatically and we don't really care what it is so no need to explicitly set it).

It's possible, I believe, for a contact to have a birthday of "8/28" (without a year), for example.